build.bat

```diff
@@ -3,4 +3,4 @@
 
 @echo off
 rem Requires a python 3.6 or higher install to be available in your PATH
-python %~dp0\tools\ci_build\build.py --build_dir %~dp0\build\Windows %*
\ No newline at end of file
+python3 %~dp0\tools\ci_build\build.py --build_dir %~dp0\build\Windows %*
```

Comment:
We don't need to change this file, this can be fixed by set Python3 as default python in your PATH.

Reply:
Done.